src/components/reviews-list/ReviewsList.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from 'react'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Avatar from '@mui/material/Avatar'
+import Divider from '@mui/material/Divider'
+import ReviewCard from '~/components/review-card/ReviewCard'
+import { styles } from '~/components/reviews-list/ReviewsList.styles.js'
+import { useTranslation } from 'react-i18next'
+
+const ReviewsList = ({ offer }) => {
+  const { t } = useTranslation()
+  const [visibleCount, setVisibleCount] = useState(1)
+  const totalReviews = offer.review.length
+
+  const showSubjectLevel = t('constant.offerCategorySubjectLevel', {
+    category: offer.category.name,
+    subject: offer.subject.name,
+    level: offer.proficiencyLevel[0]
+  })
+
+  const handleToggleReviews = () => {
+    if (visibleCount >= totalReviews) {
+      setVisibleCount((prev) => Math.max(prev - 3, 1))
```

Comment:
It would be better to move these `3` and `1` to constant in order to avoid [magic numbers](https://refactoring.guru/replace-magic-number-with-symbolic-constant) and make the code easier to read and maintain.